Makefile

```
@@ -154,6 +154,9 @@ $(BUILD_DIR)/golangci-lint:
 
 .PHONY: test-unit
 test-unit: $(BUILD_DIR) ## Run the unit tests
+	# remove all coverage files if exists
+	- rm -rf *.out
```

Is this more correct?
```suggestion
	rm -rf *.out
```